src/features/auth/screens/SignInScreen/SignInScreen.tsx

```diff
@@ -1,29 +1,175 @@
-import React, { FC, memo } from "react";
+import React, { FC, memo, useState, useEffect } from "react";
 import { AxiosResponse } from "axios";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Button from "@material-ui/core/Button";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import logo from "assets/images/logo.svg";
+import { useHistory } from "react-router-dom";
+import Snackbar from "@material-ui/core/Snackbar";
+import ForgotPassword from "./components/ForgotPassword/ForgotPassword";
+import { useStyles } from "./SignInScreen.styles";
 import { authApi } from "../../api/auth.api";
+import { setAuthentication, isAuthenticated } from "../../logic/auth.logic";
 
 const SignInScreen: FC = () => {
+  const history = useHistory();
+  const classes = useStyles();
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [rememberLogin, setRememberLogin] = useState<boolean>(false);
+  const [FPDialog, setFPDialog] = useState<boolean>(false);
+  const [errorBar, setErrorBar] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>("");
+  const authenticated = isAuthenticated();
+
+  useEffect(() => {
+    if (authenticated) history.push("/");
+  }, [authenticated, history]);
```

Comment:
this check for authentication should not be necessary, ideally we should control this in the RouterWrapper 🤔